src/hci/gnomex/controller/GetLaunchProperties.java

```diff
@@ -52,7 +52,7 @@ public void loadCommand(HttpServletRequest request, HttpSession session) {
         idCoreFacility = null;
       }
   	} catch (Exception e) {
-  		log.error(e.getClass().toString() + ": " + e);
+  		log.error(e.getClass().toString() + ": " , e);
   		e.printStackTrace();
```

Comment:
This is not necessary and generally a bad practice, especially in production code.